packages/main/src/Bar.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="{{classes.root}}"
+	design="{{design}}"
+	style="{{this.innerStyles}}"
+	role="toolbar">
+	<div class="ui5-bar-content-container ui5-bar-leftcontent-container">
+			<slot name="startContent"></slot>
+	</div>
+	<div class="ui5-bar-content-container ui5-bar-midcontent-container">
+			<slot name="middleContent"></slot>
+	</div>
+	<div class="ui5-bar-content-container ui5-bar-rightcontent-container">
```

Comment:
Not fatal, but it will look more consistent - what about changing 'ui5-bar-rightcontent-container' to 'ui5-bar-endcontent-container' since it holds endContent slot?